client/src/components/ProfileSettings/Options/Options.tsx

```diff
@@ -19,22 +30,66 @@ const Options = (): JSX.Element => {
       anchor="left"
     >
       <List>
-        <ListItem button disableGutters component={Link} to="/profile/options/edit-profile">
+        <ListItem
+          button
+          disableGutters
+          selected={selectedIndex === 0}
+          onClick={(event: React.MouseEvent<HTMLElement>) => handleListItemClick(event, 0)}
```

Comment:
In my opinion, a better way of doing this would be to use the [`useLocation` hook from `react-router-dom`](https://reactrouter.com/web/api/Hooks/uselocation) and check if the [current pathname](https://reactrouter.com/web/api/location) matches the route.

Reply:
The reason why I did this was to use the specific property: **_selected_** for the ListItem so that when you click an option it will be highlighted. However, I did use useLocation to switch between Options to render its own component accordingly. 
Highlighted Option: 
![image](https://user-images.githubusercontent.com/62577683/130504521-9ab14de0-6888-4afa-a755-2d21c92c181d.png)

Reply:
I was thinking similarly, you could use 
```
const {pathname} = useLocation();
...
    selected = { pathname === 'route-to-match' }
```
Entirely up to you though, both work 👍🏼

Reply:
Ah I see. I keep it as is for now. Thanks!